### ISSUE TITLE
Use pcapsipdump for pcap file per sip session

### ISSUE THREAD

**Issue by gvagenas** [RestComm/Restcomm-Docker#90]
Check

http://pcapsipdump.sourceforge.net/
http://www.paperstreetonline.com/2014/11/19/pcapsipdump-dump-sip-signaling-and-rtp-streams-per-call-leg-into-separate-pcap-files/

**Comment by leftyb** [comment]
@gvagenas 
Yes we need to consider an approach for sip traffic capture. Still I believe that the best approach is hommer-sip (http://sipcapture.org/).  Lets brainstorm on the tool to use a bit more.
